### DISCUSSION TITLE
[Sui 各类编程语言SDK] graph-ql版本

### DISCUSSION THREAD

**Discussion by uvd** [hoh-zone/SuiStartrek#130] [Ideas]
https://docs.sui.io/references/sui-api/beta-graph-ql

- [ ] java
- [ ] rust
- [ ] go
- [ ] swift
- [ ] python

**Comment by phanker** [comment]
我领取Java的

**Comment by cczufish** [comment]
我领取swift的

**Comment by Whitehare2023** [comment]
我领取 Python 的
